src/sentry/templates/sentry/partial/alerts.html

```diff
@@ -32,7 +32,7 @@
         {% with message.tags|split:" " as message_tags %}
           <div class="alert{% if message.tags %}{% for tag in message_tags %} alert-{{ tag }}{% endfor %}{% endif %}">
             <div class="container">
-              <a class="close" data-dismiss="alert">×</a>
+              <a class="close" onclick="this.parentElement.parentElement.remove();">&times;</a>
```

Comment:
This might get blocked by our CSP policies.

Reply:
Good catch, I’ll just reimplement the data attribute stuff